File: rximagepicker/src/main/java/com/qingmei2/rximagepicker/di/scheduler/IRxImagePickerSchedulers.java
package com.qingmei2.rximagepicker.di.scheduler;

import io.reactivex.Scheduler;

/**
 * {@link IRxImagePickerSchedulers} is used to manage the {@link Scheduler}.
 * <p>
 * Created by QingMei on 2018/1/18.
 */
public interface IRxImagePickerSchedulers {

    Scheduler ui();

    Scheduler io();
}

File: rximagepicker/src/main/java/com/qingmei2/rximagepicker/di/scheduler/IRxImagePickerSchedulers.kt
package com.qingmei2.rximagepicker.di.scheduler

import io.reactivex.Scheduler

/**
 * [IRxImagePickerSchedulers] is used to manage the [Scheduler].
 */
interface IRxImagePickerSchedulers {

    fun ui(): Scheduler

    fun io(): Scheduler
}
